readchar/key.py

```diff
@@ -71,9 +144,17 @@
     ESC + '\x5b' + '\x32' + '\x32',
     ESC + '\x5b' + '\x32' + '\x33',
     ESC + '\x5b' + '\x32' + '\x34',
+    ESC + '\x5b' + '\x32' + '\x33' + '\x7e',
+    ESC + '\x5b' + '\x32' + '\x34' + '\x7e',
     ESC + '\x4f',
     ESC + ESC,
     ESC + ESC + '\x5b',
     ESC + ESC + '\x5b' + '\x32',
     ESC + ESC + '\x5b' + '\x33',
 )
+
+
+def convertchar(charbuffer):
```

Comment:
Please, rename this to something like "char_as_string", because "converchar" says nothing.